scripts/scil_visualize_bundlesList_mosaic.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import argparse
+from distutils.version import LooseVersion
+import os
+import shutil
+
+from dipy.viz import actor, window
+import nibabel as nib
+
+if LooseVersion(nib.__version__) < LooseVersion('2.1.0'):
+    raise ImportError("Unable to import the Nibabel streamline API. "
+                      "Nibabel >= v2.1.0 is required")
+
+import numpy as np
+
+from scilpy.io.utils import (
+    add_overwrite_arg, assert_inputs_exist, assert_outputs_exists)
+
+from PIL import Image
+from PIL import ImageFont
+from PIL import ImageDraw
+
+def build_arg_parser():
+    parser = argparse.ArgumentParser(
+        description='Visualize bundles from a list of bundles. The script will output '
+                    'one screenshot per direction (6 total) per each bundles of the list.',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument('anat_reference',   help='Image used as background (e.g. T1, FA, b0)')
+    parser.add_argument('inputs',       action='store',  nargs='+',  help='List of streamline files supported by nibabel')
+    parser.add_argument('output_name',  help='Name of the output image mosaic (e.g. jpg, png)')
+    parser.add_argument('--zoom',       type=float, default=1.0, help='Rendering zoom. A value greater than 1 is a zoom-in, a value less than 1 is a zoom-out.')
+    parser.add_argument('--ttf',        type=str, default=None,  help='Path of the true type font to use for the legends')
+    parser.add_argument('--ttf_size',   type=int, default=35,    help='Font size to use for the legends')
+
+    add_overwrite_arg(parser)
+    return parser
+
+def main():
+    parser  = build_arg_parser()
+    args    = parser.parse_args()
+
+    assert_outputs_exists(parser, args, [args.output_name])
+
+    output_names = ['axial_superior', 'axial_inferior',
+                    'coronal_posterior', 'coronal_anterior',
+                    'sagittal_left', 'sagittal_right']
+
+    listOfBundles   = [f for f in args.inputs]
+    output_dir      = os.path.dirname( args.output_name )  # Where temporal files will be created 
+    # ----------------------------------------------------------------------- #
+    # Mosaic, column 0: orientation names and data description  
+    # ----------------------------------------------------------------------- #
+    width   = 300
```

Comment:
Is it my browser or is there formatting using space everywhere ?
Pep8 is usually stuff like single space between operator, no space before and after bracket 
(Run flake8 and pylint I think)

Reply:
I was using spaces everywhere. I ran flake8 and pylint and I did almost all the recommendations.